client/my-sites/plans/ecommerce-trial/ecommerce-trial-banner/index.tsx

```diff
@@ -14,7 +14,12 @@ import DoughnutChart from '../../doughnut-chart';
 
 import './style.scss';
 
-const ECommerceTrialBanner = () => {
+interface ECommerceTrialBannerProps {
+	showButton?: boolean;
+	onClick?: () => void;
+}
```

Comment:
I think the button as a whole should be passed in as a prop -- I think the interface is much simpler if we do that, as the calling context gets to define the styling and behaviour for the button. We already have a problem where the className (and thus the CSS) for the button is specific to the calling context, which is a poor pattern that's likely to break.


```suggestion
interface ECommerceTrialBannerProps {
	callToAction?: JSX.Element | null;
}
```